src/admin/storage-jobs/index.ts

```diff
@@ -0,0 +1,91 @@
+import { grpc } from "@improbable-eng/grpc-web"
+import {
+  ListStorageJobsRequest,
+  ListStorageJobsResponse,
+  StorageJobsSummaryRequest,
+  StorageJobsSummaryResponse,
+} from "@textile/grpc-powergate-client/dist/powergate/admin/v1/admin_pb"
+import { AdminServiceClient } from "@textile/grpc-powergate-client/dist/powergate/admin/v1/admin_pb_service"
+import { StorageJobsSelector } from "@textile/grpc-powergate-client/dist/powergate/user/v1/user_pb"
+import { ListSelect } from "../../storage-jobs"
+import { Config } from "../../types"
+import { promise } from "../../util"
+import { AdminListOptions } from "./types"
+
+export interface StorageJobs {
+  /**
+   * Lists StorageJobs according to the provided ListOptions.
+   * @param opts Optional ListOptions to control the behavior of listing jobs.
+   * @returns An object containing a list of storage jobs.
+   */
+  list: (opts?: AdminListOptions) => Promise<ListStorageJobsResponse.AsObject>
```

Comment:
New list api which gives historical access and different options about how to query.